pkg/gofr/cmd.go

```diff
@@ -106,6 +108,7 @@ func (cmd *cmd) handler(path string) *route {
 	return nil
 }
 
+// Option functions to configure the service
```

Comment:
can be repharased to 

`AddDescription is an Option that adds the description text for a specified subcommand.`